src/Middlewares/CacheResponse.php

```diff
@@ -71,4 +75,24 @@ protected function getReplacers(): Collection
                 return app($replacerClass);
             });
     }
+
+    protected function getLifetime(array $args): ?int
+    {
+        if (count($args) >= 1 && is_numeric($args[0])) {
+            return (int) $args[0];
+        }
+
+        return null;
+    }
+
+    protected function getTags(array $args): array
+    {
+        if (count($args) >= 1 && is_numeric($args[0])) {
+            $tags = array_slice($args, 1);
+        } else {
```

Comment:
Oh how I hate `else` statements. This can be avoid by moving `$tags = $args` to the very first line of this function.

Reply:
Good catch! I do agree.